android/local.properties

```diff
@@ -1,3 +1,3 @@
-sdk.dir=/Users/andrehaueisen/Library/Android/sdk
-flutter.sdk=/Users/andrehaueisen/Development/flutter
```

Comment:
@jensvrai you're going to want to revert this change 
@AndreHaueisen this probably should have been gitignored?